client/src/templates/Introduction/components/super-block-accordion.tsx

```diff
@@ -108,42 +119,70 @@ export const SuperBlockAccordion = ({
   return (
     <ul className='super-block-accordion'>
       {allChapters.map(chapter => {
-        if (examChallenge && chapter.name === examChallenge.chapter) {
+        const chapterAsLinkChallenge = examChallenges.find(
+          challenge => challenge.dashedName === chapter.name
```

Comment:
If we end up [adding this](github.com/freeCodeCamp/freeCodeCamp/pull/57152), these checks against the challenge dashed name won't work. Is there another way we can do this @huyenltnguyen? Like maybe adding a property in the `full-stack.json` file?

Reply:
Yeah, that makes sense @moT01, so I went with it.

I'm not 100% convinced importing directly from the curriculum is _good_, but I think it's okay for now.